owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/security/PassCodeViewModel.kt

```diff
@@ -64,6 +65,14 @@ class PassCodeViewModel(
     fun setMigrationRequired(required: Boolean) =
         preferencesProvider.putBoolean(PassCodeActivity.PREFERENCE_MIGRATION_REQUIRED, required)
 
+    fun getNumberOfAttempts() = preferencesProvider.getInt(PREFERENCE_LOCK_ATTEMPTS, 0)
+
+    fun increaseNumberOfAttempts() =
+        preferencesProvider.putInt(PREFERENCE_LOCK_ATTEMPTS, getNumberOfAttempts().plus(1))
+
+    fun resetNumberOfAttempts() =
+        preferencesProvider.putInt(PREFERENCE_LOCK_ATTEMPTS, 0)
```

Comment:
Should it be reset after logging in using biometric?

Reply:
Of course, and I will introduce it also for the pattern lock, but not finished yet 😁

Reply:
I retract my words, this won't be applied to the pattern lock for the moment. Also, it won't be reset after logging in using biometric since biometric lock won't have brute force protection for the moment neither.